### PR TITLE
[iOS] Fix clipped text in non-compact ChoiceSet

### PR DESCRIPTION
**Pull request by almedina-ms**
## Related Issue
Fixes https://github.com/microsoft/AdaptiveCards/issues/3804

## Description
Fixes height calculation for text labels in choice sets which allows text not to be clipped any more. 

## How Verified
The fix was verified manually and the test json was added to the test folder, below the result can be seen:

Simulator Screen Shot - iPhone 11 Pro Max - 2020-02-27 at 17 47 57

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/3809)

### CODE REVIEW

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSource.mm [hunk 1]
```diff
@@ -189,6 +190,13 @@ - (void)tableView:(UITableView *)tableView didDeselectRowAtIndexPath:(nonnull NS
     }
 }
 
+/*
```
Comment (nesalang) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSource.mm]: Nit: please remove this comments

**Comment by shalinijoshi19** [comment]
@almedina-Ms CI sends to be failing..can you take a look?  Tx

**Comment by shalinijoshi19** [comment]
@almedina-ms where do we increment hte version info again for the release? Can you update it to 1.2.7 whereever it is? thanks!

**Comment by shalinijoshi19** [comment]
Also @almedina-ms please create a PR directly against release/1.2 (_in addition to making hte changes in master_) where you update the version to 1.2.7 also.

**Comment by shalinijoshi19** [comment]
@almedina-ms reminder to check this in !

**Pull request merged**